packages/wdio-types/src/Options.ts

```diff
@@ -183,10 +183,7 @@ export interface WebDriver extends Connection {
     /**
      * Appium direct connect options (see: https://appiumpro.com/editions/86-connecting-directly-to-appium-hosts-in-distributed-environments)
      */
-    directConnectProtocol?: string
-    directConnectHost?: string
-    directConnectPort?: number
-    directConnectPath?: string
+    enableDirectConnect?: boolean;
```

Comment:
Can we document this option [here](https://github.com/webdriverio/webdriverio/blob/main/website/docs/Options.md#webdriver-options)?